### ISSUE TITLE
[Bug] Issues with upload-rule Kibana command

### ISSUE THREAD

**Issue by IrfanNian** [elastic/detection-rules#2062]
Problem 1:
When I run
`python3 -m detection_rules kibana --kibana-url http://localhost:5601/ -ku elastic  -kp passw0rd upload-rule -f ../detection-rules-main/rules/windows/collection_email_powershell_exchange_mailbox.toml`

An error message will appear -
`bash: passw0rd: event not found`

Problem 2: 
When i entered the password manually, without adding the -kp options, it is able to run but another error appeared -
![image](https://user-images.githubusercontent.com/63865160/176368904-d9a68e91-cfc0-4b1e-8953-b599ae36337a.png)

Any help appreciated!

**Comment by Mikaayenson** [comment]
@IrfanNian Which stack version are you using? Have you made updates to `collection_email_powershell_exchange_mailbox.toml`? Can you share the `[metadata]` section of the toml file if you have made changes?

**Comment by IrfanNian** [comment]
> @IrfanNian
> 
> * Which stack version are you using?
> * Have you made updates to `collection_email_powershell_exchange_mailbox.toml`?
> * Can you share the `[metadata]` section of the toml file if you have made changes?
> * Can you share the full error message when you supply `-kp`?

Hi,
- I'm using version 7.17 for Elasticsearch.
- No, I didn't change anything to the .toml file. Pluck out straight from this repo.
- This error message - `bash: passw0rd: event not found` is the full error message I see when supply with `- kp`.

**Comment by Mikaayenson** [comment]
TLDR; Can you try to change the `detection_rules/etc/packages.yml` from `8.4` to `7.17.3`? You will also have to pass `--replace-id` if you want to replace the uuid of the rule since it's already uploaded.

The major issue with uploading is that the logic checks to make sure that the rules can be backported to the stack that your using. If the major version does not major, the logic will raise an error as a safety precaution across major releases. 

We currently don't expose `current_version` as a CLI option to `upload-rule`.  Which may be a good update to the CLI so that users do not have to manually update the packages.yml file.

**Comment by IrfanNian** [comment]
> TLDR; Can you try to change the package `name` in `detection_rules/etc/packages.yml` from `8.4` to `7.17.3`? You will also have to pass `--replace-id` if you want to replace the uuid of the rule since it's already uploaded.
> 
> The major issue with uploading is that the logic checks to make sure that the rules can be backported to the stack that your using. If the major version does not major, the logic will raise an error as a safety precaution across major releases.
> 
> We currently don't expose `current_version` as a CLI option to `upload-rule`. Which may be a good update to the CLI so that users do not have to manually update the packages.yml file.

Oh, thanks for the info. Managed to upload successfully after after changing the version in packages.yml and adding the --replace-id option. However, at the same time, there was still an error even though it has been successfully uploaded and reflected on my Kibana - see below. 

![image](https://user-images.githubusercontent.com/63865160/176854612-e80ab820-933f-4f82-8e48-9433a064640a.png)

And also, I am still unable to use the -kp option when uploading.

**Comment by IrfanNian** [comment]
Any updates on this?
